yetanotherco/aligned_layer
# chore: add LocalStack for local s3

# How to test
1. Run anvil, operator and aggregator
2. Run docker
3. Run `make batcher_start_local`. It will use the `.env.dev` file with the local bucket credentials. Also, It will run an s3 bucket in a docker container
4. Send proofs and check everything is working
